protostar/migrator/maybe_tmp_directory.py

```diff
@@ -0,0 +1,28 @@
+from contextlib import contextmanager
+from pathlib import Path
+
+from protostar.protostar_exception import ProtostarException
+
+
+@contextmanager
+def create_maybe_tmp_directory(path: Path):
```

Comment:
```suggestion
def create_build_directory(path: Path):
```

I didn't like the `create maybe` phrase. And I think a more specific name could be better.

Reply:
Renamed to `create_output_directory`